src/components/views/right_panel/RoomHeaderButtons.tsx

```diff
@@ -45,15 +47,30 @@ const ROOM_INFO_PHASES = [
     RightPanelPhases.Room3pidMemberInfo,
 ];
 
-const PinnedMessagesHeaderButton = ({ room, isHighlighted, onClick }) => {
+type UnreadIndicatorProps = {className: string};
+
+const UnreadIndicator = ({ className }: UnreadIndicatorProps) => {
```

Comment:
props should be interfaces

```suggestion
interface IUnreadIndicatorProps {
    className: string;
}

const UnreadIndicator = ({ className }: IUnreadIndicatorProps) => {
```

Reply:
Thanks!